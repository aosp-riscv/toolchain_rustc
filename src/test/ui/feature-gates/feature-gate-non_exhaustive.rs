//#![feature(non_exhaustive)]

<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
#[non_exhaustive] //~ERROR non exhaustive is an experimental feature
=======
#[non_exhaustive] //~ERROR the `#[non_exhaustive]` attribute is an experimental feature
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
pub enum NonExhaustiveEnum {
    Unit,
    Tuple(u32),
    Struct { field: u32 }
}

fn main() { }
