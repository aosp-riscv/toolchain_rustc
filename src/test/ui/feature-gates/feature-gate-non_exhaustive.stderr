<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
error[E0658]: non exhaustive is an experimental feature
=======
error[E0658]: the `#[non_exhaustive]` attribute is an experimental feature
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
  --> $DIR/feature-gate-non_exhaustive.rs:3:1
   |
LL | #[non_exhaustive]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/44109
   = help: add `#![feature(non_exhaustive)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
